spec/high_voltage/page_spec.rb

```diff
@@ -0,0 +1,49 @@
+require "spec_helper"
+
+describe HighVoltage::Page do
+  it "produces the id for a page" do
+    expect(page(full_file_path("exists.html.erb")).id).to eq "exists"
+  end
+
+  it "produces the id for a page in a subdirectory" do
+    expect(page(full_file_path("dir/nested.html.erb")).id).to eq "dir/nested"
+  end
+
+  it "is valid for a page" do
+    expect(page(full_file_path("exists.html.erb")).valid?).to be true
```

Comment:
It's also a little hard to follow along with the number of nested method calls and their opening / closing parens. What do you think about making it easier to read with something like:

```ruby
page = page(full_file_path("exists.html.erb"))

expect(page).to be_valid
```

throughout.